docs/production-deployment/cloud/nexus/index.mdx

```diff
@@ -18,71 +18,324 @@ keywords:
   - workflow API management
 ---
 
+import { RelatedReadContainer, RelatedReadItem } from '@site/src/components/related-read/RelatedRead';
+
 :::tip SUPPORT, STABILITY, and DEPENDENCY INFO
 
-Temporal Nexus is in [Public Preview](https://docs.temporal.io/evaluate/development-production-features/release-stages#public-preview) for Temporal Cloud.
+**Temporal Nexus** is available in [Public Preview](/evaluate/development-production-features/release-stages#public-preview).
+
+:::
+
+## Using Nexus in Temporal Cloud
```

Comment:
Loving the new titles like "Build and Use Nexus Services" however, this page has become overloaded now. I'm actively breaking apart pages throughout the docs site since it increases cognitive load when trying to understand complex topics like Nexus. It also increases the perceived difficulty and learning curve of content. It doesn't help with discoverability of content and SEO. Please break this apart into sub-sections.

Reply:
Sounds good, will do.

Reply:
split this into multiple pages